pkg/crc/preflight/preflight_linux.go

```diff
@@ -191,6 +195,17 @@ func commonChecks() []Check {
 	return checks
 }
 
+func usesSystemdResolved(osRelease *linux.OsRelease) bool {
+	switch distroID(osRelease) {
+	case linux.Ubuntu:
```

Comment:
never reached because usesSystemdResolved is only called if `case linux.RHEL, linux.CentOS, linux.Fedora:`

Reply:
yes I know, it's one change less to enable it for ubuntu though. I can remove this for now if you prefer.

Reply:
I'll remove it for now in the next iteration of this PR.